tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "compileOnSave": false,
   "compilerOptions": {
+    "outDir": "./dist",
```

Comment:
I'm not sure this will work because of some pre-node-16 restrictions (I think we couldn't use `"moduleResolution": "node16"` yet). Since this is very likely to be a breaking change, we should consider going ahead with this, because I hated having to build in the root folder so export paths wouldn't break.

Reply:
Well node 14 has finally reached EOL so we could probably get away with changing that and assuming node 16 from now on. Might need to be a breaking change though.